package camera_utils.msgs;

message CameraUtilsRequest
{
    /** Type of request */
    enum Type{
        /** Acquire frame and save it to disk */
        CAPTURE         = 1;
        /** Change output directory */
        DIR             = 2;
    }

    /** Type of request */
    required Type   type        = 1;
    /** Name of the file corresponding to the acquired frame */
    optional string file_name   = 2;
    /** Path to the output folder */
    optional string output_dir  = 3;
}